src/Reflectivity/RBProgramNode.extension.st

```diff
@@ -157,7 +157,7 @@ RBProgramNode >> link: aMetaLink [
 		add: aMetaLink.
 	aMetaLink installOn: self.
 	self clearReflectivityAnnotations.
-	self methodNode compiledMethod installLink: aMetaLink.
+	(self methodNode methodClass>>self methodNode selector) installLink: aMetaLink.
```

Comment:
Maybe we could have a comment to explain why we do that to not revert this in the future? Also, is it easy to produce a regression test? If yes, it would be nice to add one (possibly in a future PR)

Reply:
I will think about a test... but it might not be that easy and I have no time before next week